tests/components/ring/test_init.py

```diff
@@ -19,50 +19,61 @@
 from homeassistant.setup import async_setup_component
 from homeassistant.util import dt as dt_util
 
-from tests.common import MockConfigEntry, async_fire_time_changed, load_fixture
+from tests.common import MockConfigEntry, async_fire_time_changed
 
 
-async def test_setup(hass: HomeAssistant, requests_mock: requests_mock.Mocker) -> None:
+async def test_setup(hass: HomeAssistant, mock_ring_client) -> None:
     """Test the setup."""
     await async_setup_component(hass, ring.DOMAIN, {})
 
-    requests_mock.post(
-        "https://oauth.ring.com/oauth/token", text=load_fixture("oauth.json", "ring")
-    )
-    requests_mock.post(
-        "https://api.ring.com/clients_api/session",
-        text=load_fixture("session.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/ring_devices",
-        text=load_fixture("devices.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/chimes/999999/health",
-        text=load_fixture("chime_health_attrs.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/doorbots/987652/health",
-        text=load_fixture("doorboot_health_attrs.json", "ring"),
-    )
+
+async def test_setup_entry(
+    hass: HomeAssistant,
+    mock_ring_client,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test setup entry."""
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
+    assert mock_config_entry.state is ConfigEntryState.LOADED
+
+
+async def test_setup_entry_device_update(
+    hass: HomeAssistant,
+    mock_ring_client,
+    mock_ring_devices,
+    freezer: FrozenDateTimeFactory,
+    mock_config_entry: MockConfigEntry,
+    caplog,
+) -> None:
+    """Test devices are updating after setup entry."""
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
+
+    front_door_doorbell = mock_ring_devices.get_device(987654)
+    front_door_doorbell.history.assert_not_called()
+    freezer.tick(SCAN_INTERVAL)
+    async_fire_time_changed(hass, dt_util.now() + timedelta(minutes=20))
```

Comment:
```suggestion
    async_fire_time_changed(hass)
```
With the freezer this is not needed

Reply:
Done